web/src/show.php

```diff
@@ -84,7 +84,8 @@
     <ul class="dupes">
         <?php foreach ($similarStudents as $index => $record): ?>
             <li>
-                <a href="?id=<?php out($record['other_id']) ?>"><?php out(chr(66 + $index)) ?></a>: <?php out(sprintf('%.0f%%', 100 * $record['weighted_mean'])) ?>
+                <a class=<?php out(chr(82 + $index))?> href="?id=<?php out($record['other_id']) ?>"><?php out(chr(66 + $index)) ?></a>: <?php out(sprintf('%.0f%%', 100 * $record['weighted_mean'])) ?>
+                <input type="checkbox" onclick="showhidetoggle(<?php out(chr(82 + $index)) ?>)" />
```

Comment:
this would result in <input type="checkbox" onclick="showhidetoggle(T)" /> but with T being undefined it would just not find any elements. You can use single quotes inside the parentheses to not collide with the double quotes of the attribute itself.